server/src/main/java/com/kritsit/casetracker/server/datalayer/UserRepository.java

```diff
@@ -120,7 +121,7 @@ public Staff getInvestigatingOfficer(String caseNumber) throws RowToModelParseEx
                 inspectors.add(getUserDetails(inspector.get("username")));
             }
             return inspectors;
-        } catch (Exception e) {
+        } catch (SQLException e) {
```

Comment:
"Exception is caught when Exception is not thrown"

This is a bit confusing but it doesn't like catching generic Exceptions because it will also catch RuntimeException.

Are you trying to catch something other than SQLException? If so, what?